ci/package.sh

```diff
@@ -145,25 +181,31 @@ do
                         # Update index yaml based on archive file name (prefer versioned archives)
                         if [ -f $assets_dir/$versioned_archive ]
                         then
-                            echo "      - id: $template_id" >> $index_src
-                            echo "        url: {{EXTERNAL_URL}}/$versioned_archive" >> $index_src
+                            if [ "$TRAVIS" == "true" ]
+                            then
+                                echo "      - id: $template_id" >> $index_src
+                                echo "        url: {{EXTERNAL_URL}}/$versioned_archive" >> $index_src
+                                echo "      - id: $template_id" >> $index_file
+                                echo "        url: $RELEASE_URL/$stack_id-v$stack_version/$versioned_archive" >> $index_file
+                            fi
 
                             if [ $rebuild_local = true ]
                             then
                                 echo "      - id: $template_id" >> $index_file_local
                                 echo "        url: file://$assets_dir/$versioned_archive" >> $index_file_local
                             fi
 
-                            echo "      - id: $template_id" >> $index_file
-                            echo "        url: $RELEASE_URL/$stack_id-v$stack_version/$versioned_archive" >> $index_file
                         elif [ -f $prefetch_dir/$versioned_archive ]
                         then
-                            # Add references to existing template archive.
-                            echo "      - id: $template_id" >> $index_src
-                            echo "        url: {{EXTERNAL_URL}}/$versioned_archive" >> $index_src
+                            if [ "$TRAVIS" == "true" ]
```

Comment:
Do not use a TRAVIS env var in package.sh. This is valid in other environments besides just TRAVIS. There is a different way to indicate whether or not certain indexes should be built. In all cases, if the repo is built, the local template should be updated. All we need here is a flag indicating whether or not we need a full release index to be generated as well, and that can be guarded by the prefetch variable (as you need the prefetch contents to generate the full index, including things that were not previously built)